posix.mak

```diff
@@ -166,7 +166,7 @@ DOC_OUTPUT_DIR:=$(PWD)/web
 W:=$(DOC_OUTPUT_DIR)
 GIT_HOME=https://github.com/dlang
 DPL_DOCS_PATH=dpl-docs
-DPL_DOCS=$(DPL_DOCS_PATH)/dpl-docs
+DPL_DOCS=$(DPL_DOCS_PATH)/dpl-docs --DRT-gcopt=parallel:0
```

Comment:
This is not the right place to add that flag. Now the `clean` target does this:
```
rm -rf ./dlang.org/web .generated dpl-docs/.dub dpl-docs/dpl-docs --DRT-gcopt=parallel:0
rm: unrecognized option '--DRT-gcopt=parallel:0'
```

Reply:
#3428